README.md

```diff
@@ -88,26 +88,22 @@ The steps required are outlined below.
 - Run the build script appropriate for your environment.
      - **Windows:** [build.cmd](https://github.com/dotnet/templating/blob/master/build.cmd)
      - **Mac/Linux**: [build.sh](https://github.com/dotnet/templating/blob/master/build.sh) 
-- When running the build script, the existing built-in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
-files with the latest Template Engine.
-- That's it! Now you can run `dotnet new3`.
+- When running the build script, the existing built-in command `dotnet new` will be preserved. To run `dotnet new3`, run `dotnet <your repo location>\artifacts\bin\dotnet-new3\Debug\netcoreapp2.1\dotnet-new3.dll` (root path to `dotnet-new3.dll` is skipped in all commands below).
```

Comment:
This is a bug. It is supposed to work with 'dotnet new3'. We now have a workaround using the full path to dotnet-new3.dll, but it should not be like that. 
1) We need to open an issue for fixing this.
2) Maybe it is better to keep the previous version with 'dotnet new3' and just add this as a temporary workaround.

Reply:
Already created #2521 to track it. I'm not sure when we have time to fix it, so I suggest once it is fixed, update readme.md again. I'm updating issue description for that.